# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
lift1:
  ID: 31
  Operating_Mode: 4
  Return_Delay_Time: 0
  Direction: 0
  Homing_Offset: -1024
  Min_Position_Limit: 1300
  Max_Position_Limit: 3800
  Goal_Position: 3000
lift2:
  ID: 32
  Operating_Mode: 4
  Return_Delay_Time: 0
  Direction: 1
  Homing_Offset: -1024
  Min_Position_Limit: 1500
  Max_Position_Limit: 4000
  Goal_Position: 1309
lift3:
  ID: 33
  Operating_Mode: 4
  Return_Delay_Time: 0
  Direction: 0
  Homing_Offset: 1024
  Min_Position_Limit: 1400
  Max_Position_Limit: 3900
  Goal_Position: 3349
lift4:
  ID: 34
  Operating_Mode: 4
  Return_Delay_Time: 0
  Direction: 1
  Homing_Offset: 1024
  Min_Position_Limit: 1500
  Max_Position_Limit: 4000
  Goal_Position: 1596

